templates/taxi/car_list.html

```diff
@@ -7,6 +8,11 @@ <h1>
       Create
     </a>
   </h1>
+  <form action="" method="get" class="form-inline">
```

Comment:
same form is added on all three templates - can you refactor to avoid repetition?